app/mixins/inplace2.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
```

Comment:
We're going to completely replace inplace, but in case we don't get it done how about we call this inplaceValidated?